common/student/screening.ts

```diff
@@ -65,3 +71,29 @@ export async function ScheduleCoCReminders(student: Student) {
     await createRemissionRequest(student);
     await Notification.actionTaken(student, 'coc_reminder', {});
 }
+
+export async function cancelCoCReminders(student: Student) {
+    const entityManager = getManager();
+    const transactionLog = getTransactionLog();
+
+    const notificationIDs = (
```

Comment:
I don't think we should replicate the notification system here. I guess it would be sufficient to just trigger another Notification.actionTaken here like 'coc_cancelled', that can then be used in the cancelOnAction on all notifications